gems/pending/postgres_ha_admin/failover_databases.rb

```diff
@@ -51,6 +51,14 @@ def query_repmgr(connection)
       result
     end
 
+    def symbolize_record_keys(record)
+      new_record = {}
+      record.each do |k, v|
+        new_record[k.to_sym] = v
+      end
+      new_record
```

Comment:
You need to group the k, v otherwise Ruby is not putting the values into the right parts...

```ruby
record.each_with_object({}) do |(k, v), hash|
  hash[k.to_sym] = v
end
```

Technically, the way .each works is that it yields 1 item to the block.  All `.each_with_object` and `.each_with_index` (or more specifically, `.each.with_object`/`.each.with_index`) do is to just expect one more param at the end...they are expecting only two params *regardless* of what .each is called on.  When .each is called on a Hash it actually yields a single Array of two elements.  When you do `|k, v, h|` without the params, the Array is going into k, the new hash is going into v, and nil is going into h.  So, adding the parens causes the array to go into that pair, which is then destructured into k and v separately.  Hope that made sense :)